wild/tests/integration_tests.rs

```diff
@@ -2425,6 +2429,66 @@ fn integration_test(
     Ok(())
 }
 
+fn external_test_suites(external_test_suites_path: String) -> Result {
+    let external_test_suites_path = PathBuf::from(external_test_suites_path);
+    if !external_test_suites_path.exists() {
+        bail!(
+            "External test suites path `{}` does not exist",
+            external_test_suites_path.display()
+        );
+    }
+
+    exec_mold_tests(external_test_suites_path)?;
+
+    Ok(())
+}
+
+fn exec_mold_tests(mold_test_path: PathBuf) -> Result {
+    let mut failed_tests = Vec::new();
+
+    let path = env::var("PATH").unwrap();
+    let current_dir = env::current_dir()?;
+    let wild_dir = current_dir.parent().unwrap();
+
+    for entry in std::fs::read_dir(&mold_test_path)? {
+        let entry = entry?;
+        if entry.file_type()?.is_file() && entry.path().extension() == Some("sh".as_ref()) {
+            let output = Command::new("bash")
+                .arg("-c")
+                .arg(entry.path())
+                .env("PATH", format!("{}:{path}", wild_dir.display()))
```

Comment:
I thought that Mold's tests relied on the current working directory containing a binary called "ld". I guess if it can't find `ld` in the current directory, it would look on the path, so this might work, although I think currently it'd find the release build of wild. Wild's repository root contains a symlink called "ld", but it's a symlink to `target/release/wild` which wouldn't be suitable if the tests were being run without `--release`. The `fakes-debug` directory contains similar symlinks, but for debug builds.

One useful manual test for your tests is to introduce some bug - say change wild so that it always panics, then run `cargo test test_external_test_suits` and verify that all of the tests now fail.